docs/src/rules/no-loop-func.md

```diff
@@ -115,6 +115,34 @@ for (let i=10; i; i--) {
     a();
 }
 
+for (const i of foo) {
+    var a = function() { return i; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
+
+for (using i of foo) {
+    var a = function() { return i; }; // OK, all references are referring to block scoped variables in the loop.
```

Comment:
I don't know why, but the indentation of these codes seems inconsistent (it might be due to mixing tabs and spaces). Could you please check it?

<img width="707" alt="image" src="https://github.com/user-attachments/assets/94972c2e-d279-4b64-b54e-d9b644509fac" />